docs/source/10min_dask_cudf.ipynb

```diff
@@ -0,0 +1,2565 @@
+      "6  6  13  6         1         1\n",
+      "7  7  12  7         0         0\n",
+      "8  8  11  8         1         0\n",
+      "9  9  10  9         0         1\n",
+      "[30 more rows]\n"
+     ]
+    }
+   ],
+   "source": [
+    "ddf = dask_cudf.read_csv('example_output/*.csv')\n",
+    "print(ddf.compute())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Parquet"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Writing to parquet files, using the CPU via PyArrow."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 74,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/conda/envs/rapids/lib/python3.7/site-packages/cudf-0.8.0a1+417.gd2833e14.dirty-py3.7-linux-x86_64.egg/cudf/io/parquet.py:56: UserWarning: Using CPU via PyArrow to write Parquet dataset, this will be GPU accelerated in the future\n",
+      "  warnings.warn(\"Using CPU via PyArrow to write Parquet dataset, this will \"\n"
+     ]
+    }
+   ],
+   "source": [
+    "df.to_parquet('example_output/temp_parquet')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Reading parquet files with a GPU-accelerated parquet reader."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 75,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "     a   b   c  agg_col1  agg_col2\n",
+      "0    0  19   0         1         1\n",
+      "1    1  18   1         0         0\n",
+      "2    2  17   2         1         0\n",
+      "3    3  16   3         0         1\n",
+      "4    4  15   4         1         0\n",
+      "5    5  14   5         0         0\n",
+      "6    6  13   6         1         1\n",
+      "7    7  12   7         0         0\n",
+      "8    8  11   8         1         0\n",
+      "9    9  10   9         0         1\n",
+      "10  10   9  10         1         0\n",
+      "11  11   8  11         0         0\n",
+      "12  12   7  12         1         1\n",
+      "13  13   6  13         0         0\n",
+      "14  14   5  14         1         0\n",
+      "15  15   4  15         0         1\n",
+      "16  16   3  16         1         0\n",
+      "17  17   2  17         0         0\n",
+      "18  18   1  18         1         1\n",
+      "19  19   0  19         0         0\n"
+     ]
+    }
+   ],
+   "source": [
+    "df = cudf.read_parquet('example_output/temp_parquet/72706b163a0d4feb949005d22146ad83.parquet')\n",
+    "print(df.to_pandas())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Writing parquet files using PyArrow under the hood."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 76,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ddf.compute().to_parquet('example_files')  "
```

Comment:
We can't use dask-cudf `to_parquet` directly?

Reply:
Good call. We can. We just can't read it back in directly with dask-cudf yet